app/src/main/java/de/dhbw/corona_world_app/ui/tools/StatisticCallRecyclerViewFragment.java

```diff
@@ -128,4 +127,23 @@ public void onStop() {
     public abstract ShowStatisticInterface getShowStatisticInterface();
 
     public abstract void initViewModelData(StatisticCallViewModel statisticCallViewModel);
+
+    protected void tryRepairingData(){
+        ErrorDialog.createBasicErrorDialog(getContext(), "Your Data seems to be corrupt", "We were not able to save new Data, we will try to fix this Problem", (dialog, which) -> {
```

Comment:
Fine error message, but as stated above, we should discuss if we replace some of it with a standardized error code/message